lumicks/pylake/tests/test_channels/test_channels.py

```diff
@@ -711,6 +711,20 @@ def test_downsampling_over_subset():
     )
 
 
+def test_downsampling_out_of_range():
+    num_points = 4
+    s = channel.Slice(channel.Continuous(np.arange(num_points), with_offset(100), 10))
+
+    for intervals in [(20, 100), (100 + 10 * num_points, 10000)]:
+        with pytest.raises(RuntimeError, match="No overlap"):
```

Comment:
I think matching to partial messages was also something that caught us in the mutation checking, no?
```suggestion
        with pytest.raises(RuntimeError, match="No overlap between slices."):
```

Reply:
It did yes, but I added ignores for those. That said, I don't mind adding it, while we're modifying this code anyhow. It should be `No overlap between range and selected channel.` though :)